Logstash/logstash-input-azureblob/lib/logstash/inputs/azureblob.rb

```diff
@@ -135,6 +206,47 @@ def process(queue)
       @logger.error("Oh My, An error occurred. \nError:#{e}:\nTrace:\n#{e.backtrace}", :exception => e)
     end # begin
   end # process
+  
+  # Get first json object out of a string, return the rest of the string
+  def get_jsons(content, batch_size)
+    return nil, content, 0 if content.nil? || content.length == 0
+    return nil, content, 0 if (content.index '{').nil?
+
+    hit = 0
+    count = 0
+    index = 0
+    first = content.index('{')
+    move_opening = true
+    move_closing = true
+    while(hit < batch_size)
+        inIndex = content.index('{', index) if move_opening
+        outIndex = content.index('}', index) if move_closing
```

Comment:
This parsing is probably not reliable.
If I have a '{' or '}' in my string values, then it will not work here. For example:

{
    "myMessage" : "I have braces like this {}"
}

Reply:
Very good point. I'll file a issue to track this / fix this issue.